<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1292</width>
    <height>807</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="choice_char_info">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>190</y>
      <width>301</width>
      <height>541</height>
     </rect>
    </property>
    <property name="title">
     <string>내 캐릭터 정보보기</string>
    </property>
    <widget class="QComboBox" name="char_cho">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>281</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QWidget" name="vwew_char" native="true">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>90</y>
       <width>261</width>
       <height>101</height>
      </rect>
     </property>
     <widget class="QLabel" name="char_cho_name">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>231</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Agency FB</family>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="lineWidth">
       <number>0</number>
      </property>
      <property name="midLineWidth">
       <number>0</number>
      </property>
      <property name="text">
       <string>캐릭터명</string>
      </property>
     </widget>
     <widget class="QLabel" name="char_cho_job">
      <property name="geometry">
       <rect>
        <x>9</x>
        <y>9</y>
        <width>241</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Agency FB</family>
        <pointsize>8</pointsize>
       </font>
      </property>
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="lineWidth">
       <number>0</number>
      </property>
      <property name="text">
       <string>직업</string>
      </property>
     </widget>
     <widget class="QLabel" name="char_cho_level">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>241</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Agency FB</family>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="lineWidth">
       <number>0</number>
      </property>
      <property name="text">
       <string>레벨</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="gem_imprint">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>200</y>
       <width>261</width>
       <height>321</height>
      </rect>
     </property>
     <property name="title">
      <string>보석 &amp; 각인</string>
     </property>
     <widget class="QListWidget" name="gam_info">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>111</width>
        <height>271</height>
       </rect>
      </property>
     </widget>
     <widget class="QListWidget" name="imprint_info">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>30</y>
        <width>111</width>
        <height>271</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QPushButton" name="char_cho_btn">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>50</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>선택하기</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="view_party">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>30</y>
      <width>281</width>
      <height>701</height>
     </rect>
    </property>
    <property name="title">
     <string>만들어진 파티</string>
    </property>
    <widget class="QListWidget" name="view_party_info">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>261</width>
       <height>601</height>
      </rect>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="horizontalScrollMode">
      <enum>QAbstractItemView::ScrollPerItem</enum>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="view_partyseat">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>30</y>
      <width>281</width>
      <height>701</height>
     </rect>
    </property>
    <property name="title">
     <string>파티자리 확인</string>
    </property>
    <widget class="QLabel" name="cho_party_title">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>241</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>선택한 파티 이름</string>
     </property>
    </widget>
    <widget class="QPushButton" name="cho_partyseat_btn">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>660</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>파티신청</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>581</y>
       <width>261</width>
       <height>71</height>
      </rect>
     </property>
     <property name="title">
      <string>신청 캐릭터 선택</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QComboBox" name="partyjoin_charcho"/>
      </item>
     </layout>
    </widget>
    <widget class="QListWidget" name="view_partyseat_info">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>256</width>
       <height>471</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="my_application">
    <property name="geometry">
     <rect>
      <x>980</x>
      <y>360</y>
      <width>281</width>
      <height>371</height>
     </rect>
    </property>
    <property name="title">
     <string>신청한 파티</string>
    </property>
    <widget class="QListWidget" name="select_party">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>256</width>
       <height>271</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="drop_select_byn">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>330</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>신청 취소</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="logininfo">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>301</width>
      <height>161</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLabel" name="hello_user">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="lineWidth">
      <number>0</number>
     </property>
     <property name="text">
      <string>어서오십셔</string>
     </property>
    </widget>
    <widget class="QLabel" name="login_id_name">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>60</y>
       <width>231</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Agency FB</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="lineWidth">
      <number>0</number>
     </property>
     <property name="text">
      <string>대표캐릭터 이름</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="my_application_2">
    <property name="geometry">
     <rect>
      <x>980</x>
      <y>30</y>
      <width>281</width>
      <height>311</height>
     </rect>
    </property>
    <property name="title">
     <string>내가 만든 파티</string>
    </property>
    <widget class="QListWidget" name="made_myparty_list">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>256</width>
       <height>211</height>
      </rect>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
    </widget>
    <widget class="QPushButton" name="drop_party_btn">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>270</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>파티 취소</string>
     </property>
    </widget>
    <widget class="QPushButton" name="making_party_btn">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>270</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>파티만들기</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1292</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>char_cho_btn</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>mychar_info()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>169</x>
     <y>270</y>
    </hint>
    <hint type="destinationlabel">
     <x>338</x>
     <y>284</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>making_party_btn</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>make_party_step()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1009</x>
     <y>333</y>
    </hint>
    <hint type="destinationlabel">
     <x>970</x>
     <y>373</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>drop_party_btn</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>drop_party()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1216</x>
     <y>333</y>
    </hint>
    <hint type="destinationlabel">
     <x>1161</x>
     <y>377</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cho_partyseat_btn</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>join_party()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>814</x>
     <y>733</y>
    </hint>
    <hint type="destinationlabel">
     <x>867</x>
     <y>777</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>drop_select_byn</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>drop_joinparty()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1226</x>
     <y>726</y>
    </hint>
    <hint type="destinationlabel">
     <x>1189</x>
     <y>775</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>view_party_info</sender>
   <signal>itemClicked(QListWidgetItem*)</signal>
   <receiver>MainWindow</receiver>
   <slot>view_partymember()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>514</x>
     <y>648</y>
    </hint>
    <hint type="destinationlabel">
     <x>614</x>
     <y>41</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>made_myparty_list</sender>
   <signal>itemClicked(QListWidgetItem*)</signal>
   <receiver>MainWindow</receiver>
   <slot>view_mypartymember()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1061</x>
     <y>191</y>
    </hint>
    <hint type="destinationlabel">
     <x>933</x>
     <y>32</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>mychar_info()</slot>
  <slot>join_party()</slot>
  <slot>make_party_step()</slot>
  <slot>drop_party()</slot>
  <slot>drop_joinparty()</slot>
  <slot>view_partymember()</slot>
  <slot>view_mypartymember()</slot>
 </slots>
</ui>
